clients/java/src/main/java/io/camunda/zeebe/client/api/response/DecisionRequirements.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.response;
+
+public interface DecisionRequirements {
+
+  /**
+   * @return the dmn decision requirements ID, as parsed during deployment; together with the
+   *     versions forms a unique identifier for a specific decision
+   */
+  String getDmnDecisionRequirementsId();
+
+  /** @return the dmn name of the decision requirements, as parsed during deployment */
+  String getDmnDecisionRequirementsName();
+
+  /** @return the assigned decision requirements version */
+  int getVersion();
+
+  /**
+   * @return the assigned decision requirements key, which acts as a unique identifier for this
+   *     decision requirements
+   */
+  long getDecisionRequirementsKey();
+
+  /**
+   * @return the resource name (see: Resource.name) from which this decision requirements was parsed
```

Comment:
:question: What is `Resource.name`?

Reply:
This was directly copied from [gateway.proto](https://github.com/camunda/zeebe/blob/main/gateway-protocol/src/main/proto/gateway.proto#L243-L245).

I'll have another look at it.